sandbox/index.html

```diff
@@ -69,8 +66,9 @@
                 </a>
                 <a href="javascript:void(null);" id="btnInspector" class="hidden"><img src="./Assets/Icon_EditModel.svg"
                         alt="Display inspector" title="Display inspector" /></a>
-                <div id="dropdownContent-env" class="hidden">
-                </div>
+                <a href="javascript:void(null);" id="animationFilesButton" class="hidden">

```

Comment:
not needed (I would like no change at all on sandbox)

Reply:
done